bin/kn-init

```diff
@@ -25,5 +25,8 @@ token="$tokenID.$tokenVal"
 
 sed -i "s/your-kubeadm-token/${token}/g" config.tfvars
 
+# Write provisioner in all config files
+sed -i "s|provisioner-image-name-autogenerated|${KN_PROVISIONERS_IMG}|g" config.tfvars*
```

Comment:
Add provisioners_img-var to config file (to allow different configs have different version of provisioner image/version tag)